### PR TITLE
Likes: to never double show on search results

### PR DESCRIPTION
**Pull request by hinaloe**
Likes button isno hide on excerpt from #484,
but... it makes dobul likes button (one is no wortk!) when enable on archives&search results.

Example
screenshot_2014-12-21-03-06-26

Fixed
screenshot_2014-12-21-03-13-22

### CODE REVIEW

**Comment by samhotchkiss** [comment]
Looks good, solid fix, thanks, @hinaloe!

File: modules/likes.php [hunk 1]
```diff
@@ -896,6 +896,9 @@ function is_likes_visible() {
 			if ( post_password_required() )
 				$enabled = false;
 
+      if ( in_array( 'get_the_excerpt', (array) $wp_current_filter ) )
```
Comment (dereksmart) [inline modules/likes.php]: This should be indented with tabs, and also use braces for the conditional please

**Comment by dereksmart** [comment]
Works great, would be happy to merge once code styles are fixed

**Comment by dereksmart** [comment]
:+1: thanks @hinaloe looks good

**Pull request merged**